src/ci/docker/host-x86_64/x86_64-gnu-nopt/Dockerfile

```
@@ -1,5 +1,7 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
 
+# Avoid interactive prompts while installing `tzdata` dependency.
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

```suggestion
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

I'd inline this to reduce the number of layers we have.

---

Fixed, thanks!